modules/imaging_browser/templates/imaging_session_controlpanel.tpl

```diff
@@ -39,7 +39,13 @@
             <li><a href="{$baseurl}/{$subject.candid}/{$subject.sessionID}/radiology_review/?commentID={$subject.RadiologyReviewCommentID}">Radiology Review</a></li>
         {/if}
         {foreach from=$subject.tarchiveids item=tarchive}
-        <li><a href="{$baseurl}/dicom_archive/viewDetails/?tarchiveID={$tarchive.TarchiveID}&backURL={$backURL|escape:"url"}">DICOM Archive {$tarchive.TarchiveID}</a></li>{/foreach}
+          <li><a href="{$baseurl}/dicom_archive/viewDetails/?tarchiveID={$tarchive}&backURL={$backURL|escape:"url"}">DICOM Archive(s) {$tarchive}</a></li>
+        {/foreach}
+        {foreach from=$subject.tarchiveidLoc item=tarchiveLoc}
+              <li><a href="/mri/jiv/get_file.php?file=tarchive/{$tarchiveLoc}" class="btn btn-primary btn-small">
```

Comment:
@christinerogers 

once this is merged (https://github.com/aces/Loris/pull/2649); and `get_file.php` is made to use the Config variable tarchiveLibDir, the hardcoded `tarchive` in `file=tarchive/{$tarchiveLoc}` can be removed